recipe/meta.yaml

```diff
@@ -1,47 +1,59 @@
 {% set name = "better-optimize" %}
 {% set version = "0.1.5" %}
-{% set python_min = "3.10" %}
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
-  url: https://pypi.org/packages/source/{{ name[0] }}/{{ name }}/better_optimize-{{ version }}.tar.gz
-  sha256: 1fa028b38b17154fe48de8d3dbd076688683b223ae9c625710e9539c9db9224c
+  url: https://github.com/jessegrabowski/{{ name.replace('-', '_')}}/archive/refs/tags/v{{ version }}.tar.gz
+  sha256: 64cddaa95ca005e9a834f59b09a63b1d20c0772d0d730b750acf3ecfe0cf06fe
 
 build:
-  noarch: python
-  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
   number: 0
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
+  skip: True  # [py<310 or py>313]
 
 requirements:
   host:
-    - python {{ python_min }}
-    - numpy
+    - pip
+    - python
+    - numpy {{ numpy }}
+    - wheel
     - setuptools
     - versioneer
```

Comment:
```suggestion
    - versioneer
    - tomli  # [py<311]
```
versioneer[toml] expands to versioneer and tomli, see https://github.com/python-versioneer/python-versioneer/blob/0.29/pyproject.toml#L2